src/renderer/coremods/settings/pages/General.tsx

```diff
@@ -134,12 +142,21 @@ export const General = (): React.ReactElement => {
               .then((answer) => {
                 if (answer) {
                   if (value) {
-                    void RepluggedNative.reactDevTools.downloadExtension().then(() => {
-                      rdtOnChange(value);
-                      setTimeout(() => window.DiscordNative.app.relaunch(), 250);
-                    });
+                    void RepluggedNative.reactDevTools
+                      .downloadExtension()
+                      .then(() => {
+                        rdtOnChange(value);
+                        relaunch();
+                      })
+                      .catch(() => {
+                        void toast.toast(
```

Comment:
Not a promise so `void` shouldn't be needed
```suggestion
                        toast.toast(
```